platform/lib/platform_web/live/media_live/edit_attribute.ex

```diff
@@ -175,6 +242,105 @@ defmodule PlatformWeb.MediaLive.EditAttribute do
                 rows={1}
                 class="!border-0 resize-none focus:ring-0 sm:text-sm shadow-none"
               />
+              <div class="md:flex items-center absolute bottom-0 right-0 mb-2 z-[100]">
+                <.live_file_input upload={@uploads.attachments} x-ref="file_input" class="sr-only" />
+                <button
+                  type="button"
+                  x-data
+                  x-on:click={"document.getElementById('#{@uploads.attachments.ref}').click()"}
+                  class="-m-2.5 w-10 h-10 rounded-full bg-white/75 backdrop-blur flex items-center justify-center text-gray-400 hover:text-gray-500"
+                >
+                  <svg
+                    class="h-5 w-5"
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 20 20"
+                    fill="currentColor"
+                    aria-hidden="true"
+                  >
+                    <path
+                      fill-rule="evenodd"
+                      d="M8 4a3 3 0 00-3 3v4a5 5 0 0010 0V7a1 1 0 112 0v4a7 7 0 11-14 0V7a5 5 0 0110 0v4a3 3 0 11-6 0V7a1 1 0 012 0v4a1 1 0 102 0V7a3 3 0 00-3-3z"
+                      clip-rule="evenodd"
+                    />
+                  </svg>
+                  <span class="sr-only">Attach a file</span>
+                </button>
+
```

Comment:
Since all this code is duplicated from `comment_box.ex`, can we extract it into its own component in `components.ex`?

Reply:
Yes, I made two components since I did not want to change the style of the comment box and edit_attribute. I hope that works